server/src/cloudify/rest.ts

```diff
@@ -0,0 +1,71 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Cloudify Platform LTD. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import fetch from 'node-fetch';
+
+// For casting the response of our Marketplace API.
+interface JSONPagination {
+	'size': number,
+	'offset': number,
+	'total': number,
+}
+export interface JSONItems {
+    [key: string]: Object;
+}
+interface JSONResponse {
+    items: JSONItems;
+	pagination: JSONPagination;
+};
+
+export function rawRequest(url:string, method:string, additionalParams={}): Promise<JSONResponse> {
+    // For getting the response from a rest service.
+	const params = {
+		...additionalParams,
+		method: method,
+		headers: {
+			Accept: 'application/json',
+		}
+	}
+	console.log('Start rawRequest %s %s', url, method)
+	const pr = fetch(url, params).then(
+		resp => {
+			return resp.json();
+		}
+	) as Promise<JSONResponse>;
+	return pr;
```

Comment:
Optional:

Can be written shorter:

```suggestion
	return fetch(url, params).then(response => response.json()) as Promise<JSONResponse>;
```